install.sh

```diff
@@ -241,18 +268,18 @@ function install_mcu_templates {
 # Step 5: restarting Klipper
 function restart_klipper {
     echo "[POST-INSTALL] Restarting Klipper..."
-    sudo systemctl restart klipper
+    sudo service $SERVICE_NAME restart
 }
 
 
-BACKUP_DIR="${BACKUP_PATH}/$(date +'%Y_%m_%d-%H%M%S')"
 
 printf "\n======================================\n"
 echo "- Klippain install and update script -"
 printf "======================================\n\n"
 
 # Run steps
 preflight_checks
+BACKUP_DIR="${BACKUP_PATH}/$(date +'%Y_%m_%d-%H%M%S')"
```

Comment:
Is it possible to add the printer name in the backup name?

Reply:
Hello, I tried to pack all changes at the same place. `BACKUP_DIR` is updated (printer name is append) at line 59.
I may propagate to [Shake&Tune](https://github.com/Frix-x/klippain-shaketune) once I do it for my CR-10.

Reply:
I discover an issue with the patch. `printer_data` is also hardcoded in `shell_commands.cfg`.  I had to manually update.
The file is a symlink from Klippain installation directory, and hence is shared across all printer configuration.

Reply:
Ok done